tests/test_syslog_server.rb

```diff
@@ -34,9 +34,15 @@ def teardown
 
   def no_syslogserver
     # Turn the feature off for a clean test.
-    config('no logging server 1.2.3.4',
-           'no logging server 2003::2',
-           'no vrf context red')
+    if platform == :ios_xr
+      config('no logging 1.2.3.4',
+             'no logging 2003::2',
```

Comment:
I get some failures in the minitest because these commands will remove 'logging 1.2.3.4 (vrf default)' but they do not remove 'logging 1.2.3.4 vrf red'. Because of the vrf-blindness of the key (see above comment) the lingering VRF logging entries cause failures:

```
Node under test:
  - name  - ios
  - type  - R-IOSXRV9000-CH
.F.FF

Finished in 36.918642s, 0.1354 runs/s, 1.1376 assertions/s.

  1) Failure:
TestSyslogServer#test_create_destroy_single_vrf_ipv6 [tests/test_syslog_server.rb:134]:
Expected {"2003::2"=>#<Cisco::SyslogServer:0x007f1666f6ec48 @name="2003::2", @level=5, @vrf="red">} to not include "2003::2".


  2) Failure:
TestSyslogServer#test_create_destroy_single_ipv6 [tests/test_syslog_server.rb:65]:
Expected {"2003::2"=>#<Cisco::SyslogServer:0x007f1665e27048 @name="2003::2", @level=5, @vrf="red">} to not include "2003::2".


  3) Failure:
TestSyslogServer#test_create_destroy_single_vrf_ipv4 [tests/test_syslog_server.rb:114]:
Expected {"1.2.3.4"=>#<Cisco::SyslogServer:0x007f16661e1378 @name="1.2.3.4", @level=4, @vrf="red">, "2003::2"=>#<Cisco::SyslogServer:0x007f16661ecae8 @name="2003::2", @level=5, @vrf="red">} to not include "1.2.3.4".

5 runs, 42 assertions, 3 failures, 0 errors, 0 skips
```